coralproject/talk
# Allow replacing comment timestamp

## What does this PR do?
New slots `commentTimestamp` and `historyCommentTimestamp` to allow replacing the comment date e.g. _10 minutes ago_ with a custom component


## How do I test this PR?

- Add components to the above slots to a new or existing plugin and see it replacing the timestamp on the comment stream and comment history.
